api/capi/src/nnstreamer-capi-util.c

```diff
@@ -85,13 +85,13 @@ ml_tensors_info_destroy (ml_tensors_info_h info)
 {
   ml_tensors_info_s *tensors_info;
 
+  if (!info)
+    return ML_ERROR_NONE;
+
   check_feature_state ();
 
   tensors_info = (ml_tensors_info_s *) info;
 
-  if (!tensors_info)
```

Comment:
can we remove this? I think this handles calling `ml_tensors_info_destroy` on the same info twice.